tdrs-backend/tdpservice/urls.py

```diff
@@ -38,6 +38,7 @@
     path("stts/", include("tdpservice.stts.urls")),
     path("data_files/", include("tdpservice.data_files.urls")),
     path("logs/", write_logs),
+    path("parsing/", include("tdpservice.search_indexes.parsers.urls")),
```

Comment:
Do we need the two layers of urls here for routing vs just importing `ParsingErrorViewSet`? Unsure on best practice here. Do you imagine we'll need another parser API in the short term? I guess #1613 could qualify too

Reply:
Normally, app has a url endpoint then subapps have their own adding on top of the app. This was we can easily tell where the endpoint code is